### PR TITLE
Update..

### PR DESCRIPTION
**Pull request by Maryom**

### CODE REVIEW

**Comment by Mazyod** [comment]
`.travis.yml` also needs to be changed to Xcode 10.2

File: .travis.yml [hunk 1]
```diff
@@ -1,13 +1,14 @@
...
-osx_image: xcode10
+osx_image: xcode10.2
 env:
   global:
   - LC_CTYPE=en_US.UTF-8
   - LANG=en_US.UTF-8
   - WORKSPACE=Appz.xcworkspace
   - IOS_FRAMEWORK_SCHEME="Appz-ios"
   matrix:
-  - DESTINATION="OS=12.0,name=iPhone SE"        SCHEME="$IOS_FRAMEWORK_SCHEME"
```
Comment (Mazyod) [inline .travis.yml:10]: Can you please revert these changes?

**Review by Mazyod — CHANGES_REQUESTED** [review]

**Comment by Mazyod** [comment]
I think it should be `12.2`

**Comment by Maryom** [comment]
What about `.swift-version` should I change it?

**Review by Mazyod — APPROVED** [review]

**Comment by Mazyod** [comment]
Not sure if it's still used, don't worry about it. Thanks ✅ 🎉

**Comment by Maryom** [comment]
What about version bump?
